### PR TITLE
HSC-24: Adding consultation,examination related concepts

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

**Comment by reagan-meant** [comment]
@rbuisson need help with fr locale proof reading of misc concepts

File: configuration/concepts/misc.csv [hunk 1]
```diff
@@ -1,26 +1,25 @@
...
-741a148e-236e-4d8f-b0ba-411befe3102a,,CIEL:1066,No,No,,Misc,N/A,,,,,,,,,,,
-26c0366b-83e4-4401-9699-e545219b8870,,CIEL:5622,Other,Other,,Misc,N/A,,,,,,,,,,,
-75e366a1-c441-4c94-9c53-312c9b8cc550,,CIEL:1175,Not Applicable,Not Applicable,,Misc,N/A,,,,,,,,,,,
-6b32bbbe-8a7c-4160-99ff-0dd6b255f458,,CIEL:1067,Unknown,Unknown,,Misc,N/A,,,,,,,,,,,
-e605731b-2e81-41a9-8446-2ed442c339e2,,CIEL:1640,Date,Date,Date,Date,,,Misc,Date,,,,,,,,,,,
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Fully specified name:fr,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1000
+0ace7699-f997-42fd-8e2e-1415d5a07c3b,,CIEL:162376,Application,Application,Application,Application,,,Misc,N/A,,,,,,,,,,,
+e6727dc7-967d-4bf6-9caf-af8a4997a62d,,CIEL:1514,Packet,Packet,Paquet,Paquet,,,Misc,N/A,,,,,,,,,,,
+144dd787-a13f-437d-b6c7-9a7d4cf3bc56,,CIEL:162398,Dose,Dose,Dose,Dose,,,Misc,N/A,,,,,,,,,,,
+f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,See comments,Voir les commentaires,Voir les commentaires,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv]: 'Commentaires'

File: configuration/concepts/misc.csv [hunk 2]
```diff
@@ -1,26 +1,25 @@
...
+9d5b708e-3f10-11e4-adec-0800271c1b75,,CIEL:162384,Dosing Units,dosing units,Unités de dosage,Unités de dosage,,,Misc,N/A,,,Capsule(s); Tablet(s); ml; mg; IU; Drop; Tablespoon; Teaspoon; Unit(s); Puff(s); Application; Packet; Dose; See comments,,,,,,,,
+0de96a2d-5c2c-4110-9770-914369de1713,,,Opthalmic,Opthalmic,Ophtalmique,Ophtalmique,,,Misc,N/A,,,,,,,,,,,
+99cbeb1b-621c-4645-a600-d702364d9885,,,Otic,Otic,Otic,Otic,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv]: "Otique"

File: configuration/concepts/misc.csv [hunk 3]
```diff
@@ -1,26 +1,25 @@
...
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginal,Intravaginale,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+9d697972-3f10-11e4-adec-0800271c1b75,,CIEL:162394,Drug Routes,Drug Routes,Routes de la drogue,Routes de la drogue,,,Misc,N/A,,,Oral; Topical; Inhalation; Intramuscular; Intravenous; Intravaginal; Opthalmic; Otic; See comments;,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv]: Voie d'administration

File: configuration/concepts/misc.csv [hunk 4]
```diff
@@ -1,26 +1,25 @@
...
+334ea16c-8a92-11e4-977f-0800271c1b75,,CIEL:160866,Q8H,Q8H,Q8H,Q8H,,,Frequency,N/A,,,,,,,,,,,
+334f851c-8a92-11e4-977f-0800271c1b75,,CIEL:160858,Q12H,Q12H,Q12H,Q12H,,,Frequency,N/A,,,,,,,,,,,
+335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+33516e22-8a92-11e4-977f-0800271c1b75,,CIEL:1099,Weekly,Weekly,Hebdomadaire,Hebdomadaire,,,Frequency,N/A,,,,,,,,,,,
+9d8e6918-3f10-11e4-adec-0800271c1b75,,CIEL:161235,Now,STAT,Maintenant,STAT,,,Frequency,N/A,,,,,,,,,,,
+2a148983-c143-4649-b4be-f8845c934ec0,,,Nightly,Nightly,Chaque nuit,Chaque nuit,,,Frequency,N/A,,,,,,,,,,,
+67d7a8d0-7879-4961-a398-63881ea7e539,,CIEL:1065,Yes,Yes,Oui,Oui,,,Misc,N/A,,,,,,,,,,,
+741a148e-236e-4d8f-b0ba-411befe3102a,,CIEL:1066,No,No,Non,Non,,,Misc,N/A,,,,,,,,,,,
+26c0366b-83e4-4401-9699-e545219b8870,,CIEL:5622,Other,Other,Autre,Autre,,,Misc,N/A,,,,,,,,,,,
+75e366a1-c441-4c94-9c53-312c9b8cc550,,CIEL:1175,Not Applicable,Not Applicable,N'est pas applicable,N'est pas applicable,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv]: Non applicable

File: configuration/concepts/misc.csv [hunk 5]
```diff
@@ -1,26 +1,25 @@
...
+6b32bbbe-8a7c-4160-99ff-0dd6b255f458,,CIEL:1067,Unknown,Unknown,Inconnue,Inconnue,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/misc.csv]: Inconnu

File: configuration/concepts/motif_consultation_exam.csv [hunk 6]
```diff
@@ -0,0 +1,13 @@
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1100
+b0823b2c-4bc8-2a7c-41b5-7bf3c7f14e82,,,Duration,Durée,Duration,Durée,,,Misc,Numeric,,,,,,,,,,,
+4f6bc54d-6168-881b-86c7-7c4f1d7a4e9e,,CIEL:1074,Months,Mois,Months,Mois,,,Misc,N/A,,,,,,,,,,,
+850c5ae8-38eb-270b-2ff2-e4ade2e95faf,,CIEL:1822,Hours,Heures,Hours,Heures,,,Misc,N/A,,,,,,,,,,,
+327cdb97-571c-2d13-306b-4024fce268cb,,CIEL:1072,Days,Jours,Days,Jours,,,Misc,N/A,,,,,,,,,,,
+1c4d0d14-8ea9-7755-3ae7-03c88e66a41a,,CIEL:1073,Weeks,Semaines,Weeks,Semaines,,,Misc,N/A,,,,,,,,,,,
+ae703c82-ed81-4b14-be4b-966949a1a696,,,Acute,Aigue,Acute,Aigue,,,Misc,N/A,,,,,,,,,,,
+9de275a5-d92c-4868-8430-973ebcc9d593,,,Chronic,Chronique,Chronic,Chronique,,,Misc,N/A,,,,,,,,,,,
+373b5749-82ff-a3d8-2e13-db76dd0499b3,,,Time Units,Unités de temps,Time lapse,Unités de temps,,,Misc,Coded,,Months;Hours;Days;Weeks,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/motif_consultation_exam.csv]: Should we replace "Time lapse" in the short name by "Time Units"?

**Review by rbuisson — CHANGES_REQUESTED** [review]
Proof reading in FR

File: configuration/concepts/misc.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/misc.csv]: Done

File: configuration/concepts/misc.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/misc.csv]: Done

File: configuration/concepts/misc.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/misc.csv]: Done

File: configuration/concepts/misc.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/misc.csv]: Done

File: configuration/concepts/misc.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/misc.csv]: Done

File: configuration/concepts/motif_consultation_exam.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/motif_consultation_exam.csv]: Done

**Review by reagan-meant — COMMENTED** [review]
Fixed requested changes

**Pull request merged**